Makefile

```diff
@@ -0,0 +1,7 @@
+default: lint test
```

Comment:
Why introduce another makefile?

Reply:
Currently etcd-watcher doesn't have any Makefile
I kinda checked the reference of this from xorm-adapter repository

Reply:
@hiteshsethi I suggest to remove this.

Reply:
Same feeling